Cargo.toml

```
@@ -18,7 +18,7 @@ repository = "https://github.com/anza-xyz/mollusk"
 readme = "README.md"
 license-file = "LICENSE"
 edition = "2021"
-version = "0.2.0"
+version = "0.2.1"
```

Don't worry about this I'll do the bumps.

```suggestion
version = "0.2.0"
```

---

Ahh might have to do this one locally to get the lockfile change.

---

merged